lib/components/Input/TextArea.tsx

```diff
@@ -71,6 +71,13 @@ export class TextArea extends React.Component<TextAreaProps, TextAreaState> {
         super(props);
     }
 
+    componentDidMount() {
+        const height = this.ghost && this.ghost.offsetHeight;
+        if (this.props.autogrow && height > 52) {
```

Comment:
why 52? where does this number come from?

Reply:
This was in the original componentDidUpdate resizing calculation - I believe it makes sure there are at least two lines of space shown so the text box doesn't get too small if there's only a little bit of content

Reply:
can you make this number a constant at the top of the file so that it can be easier to understand in the future?